azext_iot/commands.py

```diff
@@ -132,6 +132,15 @@ def load_command_table(self, _):
     ) as cmd_group:
         cmd_group.show_command("show", "iot_hub_connection_string_show")
 
+    with self.command_group(
+        "iot hub topic-space", command_type=iothub_ops
```

Comment:
The preview topic-space commands need to be attributed as `is_preview=True` prior to end user distribution.